easybuild/easyconfigs/p/pypy/pypy-7.3.5-x86-64.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'Tarball'
+
+name = 'pypy3.7'
+version = '7.3.5'
+versionsuffix = '-x86-64'
+
+homepage = 'pypy.org'
+description = """PyPy is a replacement for CPython. It is built using the RPython language that was 
+co-developed with it. The main reason to use it instead of CPython is speed: it runs generally faster
+"""
+
+toolchain = SYSTEM
+
+source_urls = ['https://downloads.python.org/pypy']
+sources = ['%(name)s-v%(version)s-linux64.tar.bz2']
```

Comment:
```suggestion
sources = ['%(name)s%(version)s-linux64.tar.bz2']
```

Reply:
And it's actually https://downloads.python.org/pypy/pypy3.7-v7.3.5-src.tar.bz2

We should NOT install binary stuff if there is a source to build from. And thus this belongs at GCCcore level together with the correct Python version as a dep?

Reply:
Building it from source is more involved compared to using the binary. But yes it is possible. And yes if we build if from source it would belong at GCCcore with Python as a dep.